BlazarUI/app/scripts/data/RepoBuildApi.js

```diff
@@ -61,20 +61,30 @@ function _getRepoBuildByParam(params, resp) {
   return findWhere(resp, {buildNumber: parseInt(params.buildNumber, 10)});
 }
 
-function fetchRepoBuild(params, cb) {
-  return _fetchBranchBuildHistory(params).then((resp) => {
+function _fetchRepoBuildId(params) {
```

Comment:
This is the extra request that we want to eventually get rid of when we update the api. Each time the page polls, it actually makes this request twice just to figure out what repoBuildId corresponds to the build number supplied in the application path.